routes/main.py

```diff
@@ -290,7 +290,10 @@ def parsefail():
         sfails = []
 
         for a in q.all():
-            tokens = json.loads(a.tokens)
+            try:
+                tokens = json.loads(a.tokens)
+            except:
+                continue
```

Comment:
`tokens = []`

Reply:
Why?

Reply:
Sorry, didn't see the context. This is fine as-is.